docs/conf.py

```diff
@@ -45,8 +48,9 @@
     pass
 
 try:
-    import sphinx
+    # third party
```

Comment:
Just out of curiosity, are these comments necessary for isort to work?

Reply:
Its not necessary to add comment headings, its optional, but in the rare situations where we do conditional imports there can be some strange issues, but generally it still marks them.